doc/source/tutorials/basictutorial.rst

```diff
@@ -0,0 +1,654 @@
+********************************
+KOS and programming introduction
+********************************
+
+This tutorial is for people who are new to programming and want to start programming using KOS.
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+
+==========================
+Accessing the KOS terminal
+==========================
+
+To use KOS you need to have a KOS processor on your vessel, they can be found under the 'control' tab in the editor and are under
+the same tab as the RCS thrusters and the reaction wheels.
+Right clicking the KOS processor in the editor after adding it to your vessel shows a screen where you can tweak settings of the
+KOS processor.
+
+There are two ways you can create KOS scripts: either you make them in game and edit them in game or you use a text editor.
+
+Making scripts in-game
+______________________
+
+First we'll look at making and editing scripts in-game.
+Put your vessel with KOS processor on the launchpad and right click the processor.
+Press the button that says 'open terminal' and click on the window that just popped up.
+
+typing the following will create a file called 'hello', to enter the command press ``ENTER`` (don't forget the period at the end):
+
+::
+
+	edit hello.
+
+
+you can now edit the file and make your script for the processor to execute, the other tutorials will show how you can make a
+working script. Keep in mind that files made in game dissapear after the vessel is gone. This happens because you locally made a file on the ship's processor.
+
+To run the file, type:
+
+::
+
+	run hello.
+
+
+Making scrips with an editor
+____________________________
+
+To use a text editor instead go to your KSP folder and go to ``/Ships/Script/`` and create an empty file and name it whatever you want
```

Comment:
Maybe explain where the KSP folder usually is, because people continue to use the wrong folders especially when using CKAN for mod installation..